docs/en/integrations/data-ingestion/data-formats/parquet-arrow-avro-orc.md

```diff
@@ -0,0 +1,325 @@
+---
+sidebar_label: Parquet, Avro, Arrow and ORC
+sidebar_position: 4
+slug: /en/integrations/data-formats/parquet-arrow-avro-orc
+---
+
+# Working with Parquet, Avro, Arrow, and ORC data in ClickHouse
+
+Apache has released multiple data formats actively used in analytics environments, including the most popular [Parquet](https://parquet.apache.org/), [Avro](https://avro.apache.org/), [Arrow](https://arrow.apache.org/), and [Orc](https://orc.apache.org/). ClickHouse supports importing and exporting data using any from that list.
+
+## Working with Parquet data
+
+Parquet is an efficient file format to store data in a column-oriented way.
+
+### Importing from Parquet
+
+Before loading data, we can use [file()](/docs/en/sql-reference/functions/files.md/#file) function to explore an [example parquet file](assets/data.parquet) structure:
+
+```sql
+DESCRIBE TABLE file('data.parquet', Parquet)
+```
+
+:::tip
+When using the `file()` function, with ClickHouse Cloud you will need to run the commands in `clickhouse client` on the machine where the file resides.
+:::
+
+We've used [Parquet](/docs/en/interfaces/formats.md/#data-format-parquet) as a second argument, so ClickHouse knows the file format. This will print columns with the types:
```

Comment:
It sounds like it's mandatory to specify format, but ClickHouse can detect format by file exctension. And as I can see, in this doc we always specify the format name while using `file`, `FROM INFILE`  and `INTO OUTFILE` when it's really unnecessary. Maybe we can change it?

Reply:
Good note. I've used explicit formats here to exclude any confusion because each article mentions multiple different formats. Also had file()/INFILE/OUTFILE links to the docs, where it says `format` is optional. Do you think this is enough, or maybe adding another small tip on this will be more useful?

Reply:
I would add a small tip on this, IMHO most readers of this doc will go and read  file()/INFILE/OUTFILE docs (I can be wrong).

Reply:
no no, you're right, 99% people won't go and read further one page, I'll add the tip

Reply:
PR: https://github.com/ClickHouse/clickhouse-docs/pull/711